docs/docs/user/migration-guide/v1-v2.md

```diff
@@ -0,0 +1,175 @@
+# Migrate from V1 to V2
+
+## [Derive component type automatically from class name](https://github.com/bakdata/kpops/pull/309)
+
+KPOps automatically inferes the `type` from the class name. Therefore, there is no need to add the `type` anymore. By convention the `type` would be the lower, and kebab cased name of the class.
```

Comment:
```suggestion
KPOps automatically infers the component `type` from the class name. Therefore, the `type` and `schema_type` attributes can be removed from your custom components. By convention the `type` would be the lower, and kebab cased name of the class.
```
TIL about the name kebab case 😂 I believe I called it dash case in the function

perhaps include the removal of `schema_type` in the diff as well

Reply:
I would like to stick with the kebab case 😄